tests/frontend/matcher/test_superglue_matcher.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the SuperGlue matcher.
+
+Authors: John Lambert
+"""
+import numpy as np
+
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.matcher.superglue_matcher import SuperGlueMatcher
+
+
+def test_SuperGlueMatcher() -> None:
```

Comment:
Superglue should inherit tests from the base matcher tests?

Reply:
So this unfortunately won't work since only 256-dimensional descriptors work with SuperGlue, while the other unit tests use descriptors of various sizes.